ui/src/containers/AppsTableContainer.js

```diff
@@ -15,8 +16,13 @@ export class AppsTableContainer extends React.Component {
     this.props.getApps();
   }
   onRowClick (event, rowId, props) {
-    console.log('On row click called');
-    this.setState({ redirect: true });
+    var app = this.props.apps.data.filter(app => {
+      return app.appName === rowId[0];
```

Comment:
Is possible more than one app has the same name? Will, it works well expected in this scenario?

Reply:
@camilamacedo86 you can't create an app with a display name that has the same as a previous one in mdc.
<img width="1059" alt="screenshot 2019-03-06 at 12 51 47" src="https://user-images.githubusercontent.com/6498727/53882666-e196ff80-400e-11e9-8167-3752e66d3d32.png">
I agree this isn't an elegant solution but we are tied somewhat with how the Table is being built in patternfly

Reply:
Usually, the solution is to hidden the info required which should not be shown to the user as follows. 

```css
.hidden {
display: none;
}

<div class="hidden"> content </div>
```